Formula/bitcoin.rb

```diff
@@ -4,7 +4,7 @@ class Bitcoin < Formula
   url "https://bitcoincore.org/bin/bitcoin-core-23.0/bitcoin-23.0.tar.gz"
   sha256 "26748bf49d6d6b4014d0fedccac46bf2bcca42e9d34b3acfd9e3467c415acc05"
   license "MIT"
-  revision 1
+  revision 2
```

Comment:
You should rebase and then update the revision to 3, as I bumped it to 2 in https://github.com/Homebrew/homebrew-core/pull/107667.

Reply:
Thanks, fixed!

Reply:
You should also squash all your commits into one.

Reply:
Thanks, squashed!